upgrade_test.py

```diff
@@ -1104,8 +1105,25 @@ def get_email_data(self):
             "grafana_screenshots": grafana_dataset.get("screenshots", []),
             "grafana_snapshots": grafana_dataset.get("snapshots", []),
             "new_scylla_repo": self.params.get("new_scylla_repo"),
-            "new_version": self.params.get("new_version"),
+            "new_version": self.new_ver,
             "scylla_ami_id": self.params.get("ami_id_db_scylla") or "-",
         })
 
         return email_data
+
+    @cache
+    def _update_argus_upgraded_version(self, node, new_version):
+        try:
+            ver = re.search(r"(?P<version>[\w.~]+)-0\.?(?P<date>[0-9]{8,8})\.(?P<commit_id>\w+)", new_version)
+            if ver:
+                package = PackageVersion(name="scylla-server-upgraded", date=ver.group("date"),
+                                         version=ver.group("version"),
+                                         revision_id=ver.group("commit_id"),
+                                         build_id=node.get_scylla_build_id() or "#NO_BUILDID")
+                self.argus_test_run.run_info.details.packages.append(package)
```

Comment:
I would also suggest updating `self.argus_test_run.run_info.details.scylla_version` field to the short version string, if going by that regex - `version` capture group, since when we filter dashboard, we use this field as a version and it's kinda confusing to see `4.6.3` or other old version on the dashboard for the enterprise-2022.1 release.